vsintegration/src/FSharp.UIResources/Strings.resx

```diff
@@ -234,4 +234,10 @@
   <data name="Show_Inline_Parameter_Name_Hints" xml:space="preserve">
     <value>Display inline parameter name hints (experimental)</value>
   </data>
+  <data name="Enable_Fast_Find_References" xml:space="preserve">
+    <value>Enable fast find references &amp; rename (experimental)</value>
```

Comment:
Now you can also add these lines [here](https://github.com/dotnet/fsharp/blob/main/vsintegration/src/FSharp.Editor/FSharp.Editor.resx#L201) - regretfully keywords are not generated automagically :(

Reply:
I guess it's not as important for experimental settings, and we'll probably just enable by default and remove them from here eventually.

Reply:
Yeah I don't know if we want people finding this just yet :)

Reply:
I'd say it's primarily for our convenience - it saves a few clicks when testing this manually and turning the switch on and off.
As you wish, of course

Reply:
Added it. Don't waste the precious clicks :)